docs/commands/mftf.md

```diff
@@ -109,6 +109,20 @@ You can include options to set configuration parameter values for your environme
 vendor/bin/mftf build:project --MAGENTO_BASE_URL=http://magento.local/ --MAGENTO_BACKEND_NAME=admin214365
 ```
 
+### `doctor`
+
+#### Description
+
+Diagnose MFTF configuration and setup
```

Comment:
Can you please add more description. 
Make it more descriptive.
Like: 
Command is checking Selenium is up and running 
Selenium available for MFTF 
Verify that new session of browser can access magento instance 
MagentoCLI command setup properly
etc...

Reply:
Fixed